packages/cardano-graphql/src/sdk.ts

```diff
@@ -380,6 +411,152 @@ export type AuthRule = {
   rule?: Maybe<Scalars['String']>;
 };
 
+export type Block = {
+  __typename?: 'Block';
+  blockNo: Scalars['Int'];
+  confirmations: Scalars['Int'];
+  epoch: Scalars['Int'];
+  epochSlot: Scalars['Int'];
+  fees: Scalars['String'];
+  hash: Scalars['String'];
+  nextBlock: Scalars['String'];
+  previousBlock: Scalars['String'];
+  size: Scalars['Int'];
+  slot: Scalars['Int'];
+  slotLeader: Scalars['String'];
+  totalOutput: Scalars['String'];
+  txCount: Scalars['Int'];
+  vrf: Scalars['String'];
+};
+
+export type BlockAggregateResult = {
```

Comment:
Some of these fields don't make sense in terms of the domain, e.g. `blockNoSum`

Reply:
Doesn't look like there's granular control over aggregate in dgraph: https://dgraph.io/docs/graphql/schema/generate